tools/osbuild-image-test

```diff
@@ -540,6 +554,18 @@ def main():
         default="manifest-db",
         help="path to the database folder"
     )
+    parser.add_argument(
+        "--instance-number",
+        type=int,
+        default=1,
+        help="if several identical instances are running specify it's number here"
+    )
+    parser.add_argument(
+        "--nb-instances",
```

Comment:
I wonder: What does `nb` mean? :)

Reply:
Ahah, that might be a French habit ? lol, It's for the total number of instances, maybe renaming it `--total-number-of-instances` might be better ? What do you prefer ?

Reply:
Do you want me to change it ? Or is  `--total-number-of-instances` good enough ?

Reply:
Well, it's quite long, but also descriptive. I don't mind either, feel to merge, or rename, I have no strong preference. :)